src/systemctl/systemctl.c

```diff
@@ -826,6 +827,9 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                                                        optarg);
 
                         if (OUTPUT_MODE_IS_JSON(arg_output)) {
+                                JsonFormatFlags *ptr = NULL;
+                                ptr = get_flag_address();
+                                *ptr = output_mode_to_json_format_flags(arg_output);
```

Comment:
Why not `arg_format_json_flags = output_mode_to_json_format_flags(...);`?

Reply:
Hello @yuwata. I actually tried doing it this way. but the challenge i have is that, the variable is being used in systemctl-show.c, it gives me the error `undeclared first use in systemctl-show.c`,  when trying to compile, when i declare it in the systemctl.c file. i can access the other variables decalred in the systemctl.c file from the systemctl-show.c file but i do not know why i can't access this one i declared myself.